src/models/language/nativeLanguageCollection.ts

```diff
@@ -11,6 +11,7 @@ export interface INativeLanguageCollection {
   css: ILanguage;
   diff: ILanguage;
   dockerfile: ILanguage;
+
```

Comment:
what does this empty line mean? I thought like a separation(because `dockerfile` is the last entry starting with a `d`) however then it would not be semantic.

Reply:
First I added dockerignore in that file but since it's not natively supported I moved it and the extra line is a leftover.